instrumentation/kamon-redis/src/main/scala/kamon/instrumentation/redisson/RedissonInstrumentation.scala

```diff
@@ -0,0 +1,68 @@
+package kamon.instrumentation.redisson
+
+import io.netty.buffer.ByteBuf
+import io.netty.channel.{ChannelFuture, ChannelFutureListener}
+import kamon.Kamon
+import kamon.trace.Span
+import kanela.agent.api.instrumentation.InstrumentationBuilder
+import kanela.agent.libs.net.bytebuddy.asm.Advice
+import org.redisson.client.protocol.{CommandData, CommandsData}
+
+import scala.collection.JavaConverters._
+
+class RedissonInstrumentation extends InstrumentationBuilder {
+
+  onType("org.redisson.client.RedisConnection")
+    .advise(method("send").and(takesArguments(1)), classOf[RedisConnectionInstrumentation])
+
+}
+
+class RedisConnectionInstrumentation
+
+object RedisConnectionInstrumentation {
+  @Advice.OnMethodEnter()
+  def enter(@Advice.Argument(0) command: Any): Span = {
+
+    def parseCommand(command: CommandData[_, _]): String = {
+      command.getParams.map {
+        case _: ByteBuf => ""
+        case bytes => String.valueOf(bytes)
+      }.filter(_.nonEmpty).mkString(" ")
+    }
+
+    val (commandName, statements) = command match {
+      case commands: CommandsData =>
+        val spanName = "redis.command.batch_execute"
+        val statements = commands.getCommands.asScala.map(parseCommand).mkString(",")
+        (spanName, statements)
+      case command: CommandData[_, _] =>
+        val spanName = s"redis.command.${command.getCommand.getName}"
+        val statement = parseCommand(command)
+        (spanName, statement)
+    }
+
+    Kamon.clientSpanBuilder(commandName, "redisson")
+      .tag("db.operation", commandName)
+      .tagMetrics("db.system", "redis")
```

Comment:
Hey @Whitilied, do you think it would be possible to ass the `db.statement` tag here with the contents of that local `statements` variable? I'm just looking at the OpenTelemetry [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#redis) (screenshot below) and thinking it would be nice to set that field as well, if we have it!

![image](https://user-images.githubusercontent.com/1302854/183074268-4cc8cf8f-3721-4c86-867a-dd730c9e9e78.png)

Reply:
gree with u

Reply:
@ivantopo change to the OpenTelemetry's Semantic Conventions